controller/cluster_check.go

```diff
@@ -116,3 +119,13 @@ func getFieldValue(v reflect.Value, name ...string) reflect.Value {
 	}
 	return getFieldValue(field, name[1:]...)
 }
+
+func debugLogsClusterExtraction(clusterName string, obj interface{}) {
+	if clusterName != "" {
+		return
+	}
+	ns := getValue(obj, "Namespace")
+	name := getValue(obj, "Name")
+	kind := getValue(obj, "Kind")
+	logrus.Debugf("failed to extract cluster that the object [%s:%s] of kind [%s] belongs to", ns, name, kind)
```

Comment:
nit: I fear that this makes it into production, and a user sees the logs and assumes that something in rancher is failing, causing them to raise a ticket.
```suggestion
	logrus.Debugf("unable to extract cluster that the object [%s:%s] of kind [%s] belongs to", ns, name, kind)
```